config/default/block.block.breadcrumbs.yml

```diff
@@ -0,0 +1,20 @@
+uuid: c1356224-451f-453d-8bbb-9394a4f49f65
+langcode: en
+status: true
+dependencies:
+  module:
+    - system
+  theme:
+    - olivero
+id: breadcrumbs
+theme: olivero
+region: header
```

Comment:
It is better to display breadcrumbs in the breadcrumbs region.
And maybe change visibility only for Commerce Product.